src/components/Divider/Divider.js

```diff
@@ -40,17 +40,22 @@ export default class Divider extends Component {
     label: PropTypes.string
   }
 
-  static defaultProps = {
-    label: "Available Options"
-  }
-
   render() {
     const { classes: { container, item, label } } = this.props;
+
+    if (this.props.label) {
+      return (
+        <div className={container}>
+          <hr className={item} />
+          <Typography component="span" className={label}>{this.props.label}</Typography>
+          <hr className={item} />
+        </div>
+      );
```

Comment:
Generally this is more duplicated identical markup than I'd like to see. A common pattern to avoid this is:

```js
return (
  <div className={container}>
    <hr className={item} />
    {!!label && <Typography component="span" className={label}>{label}</Typography>}
    <hr className={item} />
  </div>
);
```